build/tasks/release.js

```diff
@@ -49,6 +49,7 @@ gulp.task('bundle', function () {
   var bundles = [
     'app/cart/cart.component',
     'app/checkout/checkout.component',
+    'app/thank-you/thank-you.component',
```

Comment:
I noticed because this is the first AEM component that isn't going to using camel case.  I'd say to stick with camel case.  @Omicron7 do you have any thoughts?